chassis.go

```diff
@@ -32,66 +32,71 @@ type Chassis struct {
 	VtepLogicalSwitches []string
 }
 
-func (odbi *ovndb) chassisAddImp(name string, hostname string, etype string, ip string) (*OvnCommand, error) {
-	// / Prepare for encap record
-	enCapUUID, err := newRowUUID()
-	if err != nil {
-		return nil, err
-	}
-	row := make(OVNRow)
-
-	if len(name) > 0 {
-		row["chassis_name"] = name
-	}
-
+func (odbi *ovndb) chassisAddImp(name string, hostname string, etype []string, ip string) (*OvnCommand, error) {
```

Comment:
In addition, it is better to support external_ids in the API.

Reply:
Ok I will add